src/EditorFeatures/Test2/IntelliSense/CSharpCompletionCommandHandlerTests.vb

```diff
@@ -3185,97 +3177,97 @@ class C
-                Await state.WaitForAsynchronousOperationsAsync()
-                Await state.AssertCompletionSession()
-                state.SendTypeChars(".")
-                Assert.Contains("System.", state.GetLineTextFromCaretPosition())
-            End Using
-        End Function
-
-        <WpfFact, Trait(Traits.Feature, Traits.Features.Completion)>
-        Public Async Function TestNoBlockOnCompletionItems4() As Task
-            Dim tcs = New TaskCompletionSource(Of Boolean)
-            Using state = TestState.CreateCSharpTestState(
-                              <Document>
-                                  using $$
-                              </Document>, {New TaskControlledCompletionProvider(tcs.Task)})
-
-                state.Workspace.Options = state.Workspace.Options.WithChangedOption(
-                    CompletionOptions.BlockForCompletionItems, LanguageNames.CSharp, False)
-
-                state.SendTypeChars("Sys")
-                state.SendCommitUniqueCompletionListItem()
-                Await Task.Delay(250)
-                Await state.AssertNoCompletionSession(block:=False)
-                Assert.Contains("Sys", state.GetLineTextFromCaretPosition())
-                Assert.DoesNotContain("System", state.GetLineTextFromCaretPosition())
-
-                tcs.SetResult(True)
-
-                Await state.WaitForAsynchronousOperationsAsync()
-                Await state.AssertNoCompletionSession()
-                Assert.Contains("System", state.GetLineTextFromCaretPosition())
-            End Using
-        End Function
-
-        <WpfFact, Trait(Traits.Feature, Traits.Features.Completion)>
-        Public Async Function TestNoBlockOnCompletionItems3() As Task
-            Dim tcs = New TaskCompletionSource(Of Boolean)
-            Using state = TestState.CreateCSharpTestState(
-                              <Document>
-                                  using $$
-                              </Document>, {New TaskControlledCompletionProvider(tcs.Task)})
-
-                state.Workspace.Options = state.Workspace.Options.WithChangedOption(
-                    CompletionOptions.BlockForCompletionItems, LanguageNames.CSharp, False)
-
-                state.SendTypeChars("Sys")
-                state.SendCommitUniqueCompletionListItem()
-                Await Task.Delay(250)
-                Await state.AssertNoCompletionSession(block:=False)
-                Assert.Contains("Sys", state.GetLineTextFromCaretPosition())
-                Assert.DoesNotContain("System", state.GetLineTextFromCaretPosition())
-
-                state.SendTypeChars("a")
-
-                tcs.SetResult(True)
-
-                Await state.WaitForAsynchronousOperationsAsync()
-                Await state.AssertCompletionSession()
-                Assert.Contains("Sysa", state.GetLineTextFromCaretPosition())
-            End Using
-        End Function
+        '<WpfFact, Trait(Traits.Feature, Traits.Features.Completion)>
```

Comment:
❔ Same question here... how are we tracking these?

Reply:
[https://github.com/dotnet/roslyn/issues/29112](https://github.com/dotnet/roslyn/issues/29112)


---
In reply to: [208000337](https://github.com/dotnet/roslyn/pull/29016#discussion_r208000337) [](ancestors = 208000337)